lxd/main_forkproxy.go

```diff
@@ -403,7 +403,7 @@ func (c *cmdForkproxy) Run(cmd *cobra.Command, args []string) error {
 	}
 
 	// Sanity checks
-	if len(args) != 6 {
+	if len(args) < 6 || len(args) > 9 {
```

Comment:
Shouldn't that just be `!= 9` given that container_lxc.go always passes them (just setting them empty sometimes)?